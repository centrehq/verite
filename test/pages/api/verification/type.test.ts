import { createMocks } from "node-mocks-http"
import handler from "pages/api/verification/request"

describe("GET /verification/request", () => {
  it("returns the presentation definition", async () => {
    const { req, res } = createMocks({ method: "GET" })

    await handler(req, res)

    const presentation = res._getJSONData()
    expect(res.statusCode).toBe(200)
    expect(presentation.definition).toMatchSnapshot()
  })
})
