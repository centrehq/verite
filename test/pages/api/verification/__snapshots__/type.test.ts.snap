// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /verification/request returns the presentation definition 1`] = `undefined`;
